Header header
int32 sequ
int8 fsrbk
int8 fsrfl
int8 fsrfr
float32 ir
float32 prox
